src/tasks/semantic-release/index.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const fs = require('fs');
+const { MrmError, packageJson, lines, yaml, markdown, uninstall } = require('mrm-core');
+
+const PACKAGE_NAME = 'semantic-release';
+
+function hasSemanticRelease() {
+	const commands = yaml('.travis.yml').get('after_success');
+	return commands && commands.find(cmd => cmd.includes(PACKAGE_NAME)).length > 0;
+}
+
+module.exports = function(config) {
+	// Require .travis.yml
+	if (!fs.existsSync('.travis.yml')) {
+		throw new MrmError(
+			`Run travis task first:
+  mrm travis`
+		);
+	}
+
+	// package.json
+	const pkg = packageJson('package.json');
+
+	if (!hasSemanticRelease()) {
+		throw new MrmError(
+			`Setup semantic-release first:
+  npm i -g semantic-release-cli@latest && semantic-release-cli setup
+  
+semantic-release needs to add required auth keys to Travis CI.
+WARNING: Do not agree to update your .travis.yml when asked.
+
+More info:
+https://github.com/semantic-release/semantic-release#setup
+`
+		);
+	}
+
+	// Remove semantic-release devDependency
+	if (pkg.get(`devDependencies.${PACKAGE_NAME}`)) {
+		pkg.unset(`devDependencies.${PACKAGE_NAME}`);
+	}
+
+	// Remove semantic-release script
+	if (pkg.getScript(PACKAGE_NAME)) {
+		pkg.removeScript(PACKAGE_NAME);
+	}
+
+	// Save package.json
+	pkg.save();
+
+	yaml('.travis.yml')
+		// Remove the official semantic-release runner from commands list on .travis.yml
+		.set(
+			'after_success',
+			yaml('.travis.yml').get('after_success').filter(cmd => cmd !== 'npm run semantic-release')
+		)
+		// Add global semantic-release runner to .travis.yml
+		.merge({
+			after_success: [
+				`npm install -g semantic-release`,
+				'semantic-release pre && npm publish && semantic-release post',
+			],
+			branches: {
+				except: ['/^v\\d+\\.\\d+\\.\\d+$/'],
+			},
+		})
+		.save();
+
+	// Add Changelog.md to .gitignore
+	lines('.gitignore').add('Changelog.md').save();
```

Comment:
I’d make file name configurable like `Readme.md` below. This is actually a questionable step because it’s part of my release workflow and probably won’t work for other people.

Reply:
Should I remove the step you think?

Reply:
Yeah, I think I’ll just make gitignore task extendable and add it there in my own config. Let’s try to make it more generic so more than two people could use it ;-)

Reply:
Well, I think switching to semantic-release should mean dropping the `Changelog.md` so I'd even add a deletion of it to this task. How many people who use semantic-release do also support Changelog.md?

Reply:
Some projects keep a link to the Releases page there so I’m not 100% sure what’s the best approach here.

Reply:
I'd try to solve our own problems and wait for people to ask additional features as they start using it. But it's your call. Is it possible to add a text to `Changelog.md` as a first line?

Reply:
Then I’d make a file name configurable and just get rid of it as an MVP ;-)

Reply:
Done already :) 

https://github.com/sapegin/mrm/pull/16/files#diff-1ee1dbb55348d75979a5b82757c41df2R68